app/services/migration/migrators/user.rb

```diff
@@ -52,11 +56,30 @@ def unique_validated_trns(ecf_user)
         .uniq
     end
 
+    def unique_emails(ecf_user)
+      ecf_user.npq_applications.map { |app| app.participant_identity.email.to_s.downcase.strip }.compact.uniq
```

Comment:
could we include participant identities from participant profiles as well, just incase a participant profile has a different identity to an NPQ application?